### PR TITLE
Update changelog to prepare for 2.79 release

### PR DESCRIPTION
**Pull request by car-roll**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ## Changelog
 
+### 2.79
+
+Release date: 2020-02-14
+
+* Fix: Allow main script bindings to be mapped to the shell upon deserialization. Previously, if the main script had a binding that could not be deserialized, it would get silently dropped in loaded scripts.
+* Introduced GroovySample extension point
```
Comment (dwnusbaum) [inline CHANGELOG.md]:
```suggestion
* Developer: Introduced `GroovySample` extension point to allow plugins to dynamically add samples to (or filter samples from) the dropdown menu shown when configuring a Pipeline script in Jenkins. ([PR 350](https://github.com/jenkinsci/workflow-cps-plugin/pull/350))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 ## Changelog
 
+### 2.79
```
Comment (dwnusbaum) [inline CHANGELOG.md]: You'll actually be releasing 2.80. Sorry for the confusion, now that we aren't using the wiki for changelogs, I keep forgetting that we need to add release notes for security fixes here ourselves! Could you add this content for 2.79?

```
### 2.79

Release date: 2020-02-12

* Security: Fix sandbox bypass vulnerability. ([SECURITY-1710](https://jenkins.io/security/advisory/2020-02-12/#SECURITY-1710))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+
+Release date: 2020-02-14
+
+* Fix: Allow main script bindings to be mapped to the shell upon deserialization. Previously, if the main script had a binding that could not be deserialized, it would get silently dropped in loaded scripts.
```
[the remark this answers is not in the document]
Comment (car-roll) [reply CHANGELOG.md]: thanks, totally forgot about adding PR links

[the remark this answers is not in the document]
Comment (car-roll) [reply CHANGELOG.md]: addressed in 092c181

File: CHANGELOG.md (hunk 1 above)
Comment (car-roll) [reply CHANGELOG.md]: addressed in 092c181

File: CHANGELOG.md (hunk 2 above)
Comment (car-roll) [reply CHANGELOG.md]: addressed in 092c181

**Pull request merged**